projects/plugins/jetpack/extensions/blocks/dialogue/components/participants-control.js

```diff
@@ -5,49 +5,28 @@ import {
 	DropdownMenu,
 	MenuGroup,
 	MenuItem,
-	TextControl,
 	SelectControl,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
-function ParticipantsMenu( { participants, className, onSelect } ) {
+function ParticipantsMenu( { participants, className, onSelect, participantSlug } ) {
 	return (
 		<MenuGroup className={ `${ className }__participants-selector` }>
-			{ participants.map( ( { participant, participantSlug } ) => (
-				<MenuItem key={ participantSlug } onClick={ () => onSelect( { participantSlug } ) }>
+			{ participants.map( ( { participant, participantSlug: slug } ) => (
+				<MenuItem
+					key={ slug }
+					onClick={ () => onSelect( { participantSlug: slug } ) }
```

Comment:
The drop-down menu is staying open after a menu item is clicked. Should it close now that there isn't a text input?

Reply:
Yes, I think so. It would be a nice improvement. Addressed here 51ee29f11a61dcbb98a71f94da41c09b62db5a09